README: Add build instructions

rnestler opened the pull request:
I'm not sure if qtcreator is really needed. maybe one can get along with qmake and make. Also I stole the screen-shot from http://stackoverflow.com/questions/8860712/setting-default-make-options-for-qt-creator maybe we want to create our own.

ubruhin commented:
- building from command line is shown in .travis.yml
- maybe replace the screenshot with the link to stackoverflow?

rnestler commented:
So I updated and rebased the pull request:
 - show building from command line
 - replace screenshot with link to stackoverflow
Maybe we should also add a note about running the executable and tests?
